assets/js/admin/ppec-upgrade-notice.js

```diff
@@ -0,0 +1,43 @@
+;(function ( $, window, document ) {
+	'use strict';
+
+	// Check whether PayPal Payments is installed.
+	let is_paypal_payments_installed = false,
+		is_paypal_payments_active = false;
+
+	const targetElement = $( 'tr[data-slug="woocommerce-paypal-payments"]' );
```

Comment:
Even though it is unlikely, it could happen that a user has renamed the PayPal Payments directory to something else (say `woocommerce-paypal-payments2/`) for some reason.
Given the PHP uses `woocommerce-paypal-payments/woocommerce-paypal-payments.php` when generating the activation button we should probably look for a `data-plugin` attribute with that same value instead of relying on `data-slug`. That way, we can leave possible user plugin modifications undisturbed by installing/activating only the official (vanilla) extension.

To test:

1. Install PayPal Payments but keep it disabled.
2. Rename its directory.
3. Click "Activate PayPal Payments now" inside the PPEC message.
4. See a "Plugin file does not exist" error.

What are your thoughts?

Reply:
Good catch @jorgeatorres. The easiest fix I can come up with is to update the activation URL on notice via JS by copying the original plugin activation URL.